.drone.yml

```diff
@@ -122,7 +122,7 @@ steps:
       - docker buildx create --name builder --driver docker-container --use
       - docker buildx inspect --bootstrap --builder builder
       - docker buildx build
-        --platform linux/amd64,linux/arm/v7,linux/arm64/v8
+        --platform linux/amd64,linux/arm/v7,linux/arm64
```

Comment:
Is there any reason for this change?

Reply:
Not particularly, instead should I include v6 as well?